corehq/apps/app_manager/templates/app_manager/v2/partials/case_list.html

```diff
@@ -62,7 +63,7 @@ <h4 class="panel-title panel-title-nolink">
             </div>
         </div>
         {% endif %}
-        <div data-bind="visible: useCaseTiles() == 'yes' || COMMCAREHQ.toggleEnabled('CASE_LIST_CUSTOM_XML')">
+        <div data-bind="visible: (useCaseTiles() == 'yes' && COMMCAREHQ.toggleEnabled('CASE_LIST_TILE')) || COMMCAREHQ.toggleEnabled('CASE_LIST_CUSTOM_XML')">
```

Comment:
Yup! its not a part of the feature that i am adding but an improvement in what is present

the reason
```
the CASE_LIST_TILE toggle allows user to set useCaseTiles property. If someone sets that to 'yes' and then the toggle is removed for the domain, the container below is still visible since useCaseTiles was set to yes.
```